src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -289,7 +292,7 @@ private Bucket newCloudBucket(GoogleBucketResource bucketResource) {
             //.setRequesterPays()
             // See here for possible values: http://g.co/cloud/storage/docs/storage-classes
             .setStorageClass(StorageClass.REGIONAL)
-            .setLocation(gcsConfiguration.getRegion())
```

Comment:
This was the only place that was using the `gcsConfiguration` in this class so we could remove the field as well.